Blog: How to craft product survey questions

ivanagas opened the pull request:
## Changes

A blog on how to craft product survey questions with lots of examples. Would appreciate feedback on:

- Any more example questions, categories, or real companies I should add?
- Should I add a better outro? Previously I had "how to utilize responses"
- Any thoughts on title?

## Checklist
- [x] Titles are in [sentence case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case)
- [x] Feature names are in **[sentence case too]([https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case))**
- [x] Words are spelled using American English
- [x] I have checked out our [style guide](https://github.com/PostHog/posthog.com/blob/master/STYLEGUIDE.md)

ivanagas commented:
Feel free to give high-level feedback on this one, it might not be ready for line edits.

andyvan-ph commented:
Looking good so far. Graphic is ace.

Some thoughts:
- I think it would be worth doing a section on using surveys to measure / find product-market fit.
- It definitely needs some kind of conclusion – maybe a recap, or some best practices processes to follow?
- On headline, maybe "How to write effective product survey questions (with examples)". Looking at search results, most are 14+ xyz product survery questions etc. We can workshop headlines later.
- Some of the example questions are bit long-winded. Could be reworded.
- I'd be tempted to add more examples, too, just to add to the sense of thoroughness. I think we could use these questions as the basis for a newsletter, too.

joethreepwood commented:
Yep, agree this is looking good. 

High-level: 
- I think it's worth covering something around identifying users. Basically recommending that you use a survey platform where users don't need to enter name, email, etc. 
- Could be worth adding something around follow-ups. E.g. Tell users if you want to follow up with them, or ask for permission. 
- Agree with Andy about the title needing work
- Conclusion needs work and should probably lead into how to turn feedback into actionable snapshots

Line edits I didn't add: 
- Some examples of passive voice 
- We could add a link to our NPS tutorial 
- Include some screenshots of our survey tool or the example surveys where relevant

ivanagas commented:
Ok addressed all your comments
- Added product-market fit section
- Changed title
- Added conclusion
- Reworded and added some questions
- Added a bit about targeting users
- Added a screenshot

joethreepwood commented on contents/blog/product-survey-questions.md:
```diff
@@ -0,0 +1,162 @@
+
+To help you do this, this post goes over the importance of goals for product survey questions, the principles of great questions, question examples, the question real companies ask, and how to use responses. 
+
+> **Want to run surveys?** See our tutorial on [how to create custom surveys](/tutorials/survey) with options like open text, multiple choice, or custom components.
```
```suggestion
> **Want to run surveys?** See our tutorial on [how to create custom surveys](/tutorials/survey) with options such as open text, multiple choice, or custom components.
```

joethreepwood commented on contents/blog/product-survey-questions.md:
```diff
@@ -0,0 +1,162 @@
+
+Because surveys provide open, qualitative feedback from users, goals don’t need to be specific. For example, they don’t need to be a single metric you aim to improve. The goal of an individual survey should align with broader goals for a product and ultimately work towards improving it.
+
+Having a goal enables you to decide how the survey runs and who it targets. For example, if the goal is to improve the onboarding flow, it should run soon after users complete the flow. It shouldn’t run for long-time users who completely forget about the onboarding process or went through an entirely different process.
```
```suggestion
A goal enables you to decide how the survey runs and who it targets. For example, if the goal is to improve the onboarding flow, it should run soon after users complete the flow. It shouldn’t run for long-time users who completely forget about the onboarding process or went through an entirely different process.
```

joethreepwood commented on contents/blog/product-survey-questions.md:
```diff
@@ -0,0 +1,162 @@
+
+Like an [A/B test](/blog/ab-testing-guide-for-engineers), you should start your product survey process with a goal in mind. A product survey, by definition, has a goal related to the product. This could include improving a feature, prioritizing feature futures, or optimizing critical flows.
+
+Because surveys provide open, qualitative feedback from users, goals don’t need to be specific. For example, they don’t need to be a single metric you aim to improve. The goal of an individual survey should align with broader goals for a product and ultimately work towards improving it.
```
```suggestion
Goals don’t need to be specific. For example, they don’t need to be a single metric you aim to improve. The goal of an individual survey should align with broader goals for a product and ultimately work towards improving it.
```

joethreepwood commented on contents/blog/product-survey-questions.md:
```diff
@@ -0,0 +1,162 @@
+
+- Stats show you what is happening, but surveys help you understand why it is happening. 
+
+- Stats are often lagging indicators of product-market fit, they begin improve after you have it. Surveys are leading indicators, feedback will hint that you have it.
```
```suggestion
- Stats are often lagging indicators of product-market fit, they improve after you have it. Surveys are leading indicators, feedback will hint that you have it.
```

joethreepwood reviewed (approved):
lgtm

andyvan-ph commented on contents/blog/product-survey-questions.md:
```diff
@@ -0,0 +1,162 @@
+
+Once you [implement your product survey](/tutorials/survey) and get all the data from it, you need to put it to work. Asking questions and collecting responses is useless if you don’t do anything with them.
+
+First, you can use product survey data as a jumping-off point for more research. In PostHog, you can easily connect respondents to their usage data or look into usage trends based on responses. You can also use the responses to identify [users for interviews](/tutorials/feedback-interviews-site-apps) and dig into details further.
```
```suggestion
1. **Do more research** In PostHog, it's easy to connect respondents to their usage data, or look into usage trends based on responses. You can also use the responses to identify [users for interviews](/tutorials/feedback-interviews-site-apps) and dig into details further.
```

andyvan-ph commented on contents/blog/product-survey-questions.md:
```diff
@@ -0,0 +1,162 @@
+
+First, you can use product survey data as a jumping-off point for more research. In PostHog, you can easily connect respondents to their usage data or look into usage trends based on responses. You can also use the responses to identify [users for interviews](/tutorials/feedback-interviews-site-apps) and dig into details further.
+
+Second, you can monitor satisfaction with your product and features. Running repeated surveys asking the same questions can give you an idea of user perceptions over time. You can use this to track the impact of changes and identify areas requiring change.
```
```suggestion
2. **Start monitoring customer satisfaction** – Run repeated surveys asking the same questions to give you an idea of user perceptions over time. You can use this to track the impact of changes and identify areas requiring change.
```

andyvan-ph commented on contents/blog/product-survey-questions.md:
```diff
@@ -0,0 +1,162 @@
+
+Second, you can monitor satisfaction with your product and features. Running repeated surveys asking the same questions can give you an idea of user perceptions over time. You can use this to track the impact of changes and identify areas requiring change.
+
+Third and finally, you can prioritize future development and make data-driven decisions. Ultimately, the goal of product tools (and PostHog) is to help you build better products. Knowing what users think about your product and what they want to see next is a great way to do that.
```
```suggestion
3. **Update your product roadmap** – Finally, you can prioritize future development and make data-driven decisions. Ultimately, the goal of product tools (and PostHog) is to help you build better products. Knowing what users think about your product and what they want to see next is a great way to do that.
```

andyvan-ph commented on contents/blog/product-survey-questions.md:
```diff
@@ -0,0 +1,162 @@
+Second, you can monitor satisfaction with your product and features. Running repeated surveys asking the same questions can give you an idea of user perceptions over time. You can use this to track the impact of changes and identify areas requiring change.
+
+Third and finally, you can prioritize future development and make data-driven decisions. Ultimately, the goal of product tools (and PostHog) is to help you build better products. Knowing what users think about your product and what they want to see next is a great way to do that.
+
```
```suggestion

As an all-in-one platform, PostHog is uniquely placed to do all the above. In addition to surveys, we also do product analytics, session replay, feature flags, and A/B testing. If this sounds useful to you, [check us out](https://posthog.com/).
```

andyvan-ph commented on contents/blog/product-survey-questions.md:
```diff
@@ -0,0 +1,162 @@
+
+## Utilizing your product survey results
+
+Once you [implement your product survey](/tutorials/survey) and get all the data from it, you need to put it to work. Asking questions and collecting responses is useless if you don’t do anything with them.
```
```suggestion
Once you [implement your product survey](/tutorials/survey) and get all the data from it, you need to put it to work. You can: 
```

andyvan-ph commented on contents/blog/product-survey-questions.md:
```diff
@@ -0,0 +1,162 @@
+
+- What is missing from our product?
+- How can we improve feature X?
+- What prevents you from achieving your goal with our product?
```
Is this leading, maybe? I dunno.

andyvan-ph commented on contents/blog/product-survey-questions.md:
```diff
@@ -0,0 +1,162 @@
+- How useful would X feature be to you?
+- What new features do you want to see the most?
+
+### 4. Competitors
```
Comparative feels easier to parse for me

andyvan-ph reviewed (commented):
This is looking good. Have done line edits and left suggestions.

Still need to figure out the headline – this is probably too similar to all the others? – but need recharge the brain before I think more about it.

andyvan-ph commented on contents/blog/product-survey-questions.md:
```diff
@@ -0,0 +1,167 @@
+---
+title: 18 product survey questions to help build a great product
```
```suggestion
title: "How to write great product survey questions (with examples)"
```

Thoughts on his headline, @ivanagas?

ivanagas replied:
Good with me
